src/components/PGN.vue

```diff
@@ -201,6 +210,10 @@ export default {
         commentsObj[el.fen] = el.comment;
       });
       return commentsObj;
+    },
+    flipBoard() {
+      this.isBoardFlipped = !this.isBoardFlipped;
+      this.loadPgn();
     }
   }
 };
```

Comment:
In the style tag - Could you update the width of the .btn-flex class to allow more space for the buttons - maybe something like 400px instead of 300px?